src/app/app.module.ts

```diff
@@ -36,6 +37,7 @@ const appRoutes: Routes = [
     { path: 'admin-dashboard/review', component: ReviewComponent },
     { path: 'teacher-dashboard/review', component: ReviewComponent },
     { path: 'student-dashboard/review', component: ReviewComponent },
+    { path: 'admin-dashboard/create/:_id', component: AdminCreateComponent},
```

Comment:
if you get rid of the id from admin-dashboard.component it can be removed from here as well